fixed broken matplotlib link

Daniel-Nwaeze opened the pull request.

duckontheweb commented on Lecture 6/README.md:
```diff
@@ -21,4 +21,4 @@ To learn more about the dataset, checkout its [registry page](http://registry.ml
 
-2) [BigEarthNet](http://registry.mlhub.earth/10.14279/depositonce-10149/): This is a land cover classification dataset, and it contains ~570,000 images with multiclass labels for each image. This is a very large dataset, and if you are running it on a local computer, you may need to select a subset of the data for training. 
\ No newline at end of file
+2) [BigEarthNet](http://registry.mlhub.earth/10.14279/depositonce-10149/): This is a land cover classification dataset, and it contains ~570,000 images with multiclass labels for each image. This is a very large dataset, and if you are running it on a local computer, you may need to select a subset of the data for training. 
```
@Daniel-Nwaeze Was there actually a change here? I can't see anything different between the two.

Daniel-Nwaeze commented:
Never mind, Jon. Fixed it already

On Wed, 17 Nov 2021, 3:40 pm Daniel Nwaeze, ***@***.***> wrote:

> The previous link for matplotlib was broken, I replaced it
>
> On Wed, 17 Nov 2021, 3:30 pm Jon Duckworth, ***@***.***>
> wrote:
>
>> ***@***.**** commented on this pull request.
>> ------------------------------
>>
>> In Lecture 6/README.md
>> <https://github.com/RadiantMLHub/ml4eo-bootcamp-2021/pull/29#discussion_r751296717>
>> :
>>
>> > -2) [BigEarthNet](http://registry.mlhub.earth/10.14279/depositonce-10149/): This is a land cover classification dataset, and it contains ~570,000 images with multiclass labels for each image. This is a very large dataset, and if you are running it on a local computer, you may need to select a subset of the data for training.
>> \ No newline at end of file
>> +2) [BigEarthNet](http://registry.mlhub.earth/10.14279/depositonce-10149/): This is a land cover classification dataset, and it contains ~570,000 images with multiclass labels for each image. This is a very large dataset, and if you are running it on a local computer, you may need to select a subset of the data for training.
>>
>> @Daniel-Nwaeze <https://github.com/Daniel-Nwaeze> Was there actually a
>> change here? I can't see anything different between the two.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/RadiantMLHub/ml4eo-bootcamp-2021/pull/29#pullrequestreview-808666303>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AWPVPZCSZZ4RYVNRVUETWEDUMO37VANCNFSM5IGTP4YQ>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>
